flatpak/manifest_tool/flutter/rust.py

```
@@ -0,0 +1,167 @@
+    rust_bin = "/usr/lib/sdk/rust-stable/bin"
+    rustup_bin = "/run/build/lotti/.cargo/bin"
+
+    for module in modules:
+        if not isinstance(module, dict) or module.get("name") != "lotti":
+            continue
+
+        build_options = module.setdefault("build-options", {})
+
+        # Update append-path
+        if _update_append_path(build_options, rust_bin, rustup_bin):
+            messages.append(f"Added Rust paths to append-path")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)